shanoir-ng-datasets/src/main/java/org/shanoir/ng/dicom/web/dto/mapper/ExaminationToStudyDTOMapper.java

```diff
@@ -0,0 +1,112 @@
+package org.shanoir.ng.dicom.web.dto.mapper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.mapstruct.Mapper;
+import org.shanoir.ng.anonymization.uid.generation.UIDGeneration;
+import org.shanoir.ng.dataset.model.Dataset;
+import org.shanoir.ng.datasetacquisition.model.DatasetAcquisition;
+import org.shanoir.ng.dicom.web.dto.InstanceDTO;
+import org.shanoir.ng.dicom.web.dto.MetadataDTO;
+import org.shanoir.ng.dicom.web.dto.SerieDTO;
+import org.shanoir.ng.dicom.web.dto.StudyDTO;
+import org.shanoir.ng.examination.model.Examination;
+import org.shanoir.ng.shared.model.Subject;
+import org.shanoir.ng.shared.repository.SubjectRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * This class maps the Examination objects from the Shanoir-NG database
+ * to a DICOM-study DTO to implement the DICOMweb protocol.
+ * 
+ * @author mkain
+ *
+ */
+@Mapper(componentModel = "spring")
+public abstract class ExaminationToStudyDTOMapper {
+	
+	@Autowired
+	protected SubjectRepository subjectRepository;
+	
+	public StudyDTO examinationToStudyDTO(Examination examination) {
+		StudyDTO studyDTO = new StudyDTO();
+		final String studyInstanceUID = UIDGeneration.ROOT + ".1." + examination.getId();
+		// 5 DICOM-study specific values
+		studyDTO.setStudyInstanceUID(studyInstanceUID);
+		studyDTO.setStudyID(examination.getId());
+		studyDTO.setStudyDescription(examination.getComment());
+//		studyDTO.setStudyDate(examination.getExaminationDate().toString());
```

Comment:
Why don't we keep exam date here ?

Reply:
- as stated in my JavaDoc please ignore DICOMJson, as it is only a draft, that might be interesting for v3 of OHIF-viewer, the issue here is: the shanoir exam date format does not corrspond to DICOM date format